SpringProject/src/main/java/com/gradproject2019/Controller.java

```diff
@@ -1,13 +1,20 @@
 package com.gradproject2019;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+
 
 @org.springframework.stereotype.Controller
-@RequestMapping(path="/all")
+//@RequestMapping(path="/all")
 public class Controller {
 
     @Autowired
     private ConferenceRepository conferenceRepository;
 
-}
+    @GetMapping(path="/ListOfConferences")
```

Comment:
I don't think that's a very good path name. REST suggests paths that describe resources, so maybe something like 
/conferences/all or conferences/list.

Then later one you could do something like conferences/conference?id=123 if you're looking for individual conferences

Reply:
tbh I do not think the /all or /list is needed in this case